### PR TITLE
Topic detail pane placeholder

### PR DESCRIPTION
**Pull request by jdkoren**
Adds placeholder UI for the topic detail pane on the Interests tab when no topic has been selected yet.

### CODE REVIEW

**Comment by jdkoren** [comment]
Screenshot_20240313_143751

File: feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.google.samples.apps.nowinandroid.core.designsystem.theme.NiaTheme
+
+@Composable
+fun TopicDetailPlaceholder() {
```
Comment (alexvanyo) [inline feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt:38]: Nit: This should take a `modifier: Modifier = Modifier` and pass it to the `Card`.

**Review by alexvanyo — APPROVED** [review]

File: feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    Card(
+        colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.surfaceVariant),
+        shape = RoundedCornerShape(24.dp, 24.dp, 0.dp, 0.dp),
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
```
[the remark this answers is not in the document]
Comment (Jaehwa-Noh) [reply feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt:44]: I see. I missed that point.
Thank you for answering.

**Review by dturner — APPROVED** [review]

File: feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt (hunk 1 above)
Comment (jdkoren) [reply feature/topic/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/topic/TopicDetailPlaceholder.kt]: Done

**Pull request merged**